content/about/about-content.md

```diff
@@ -2,26 +2,46 @@
 
 <!--The text at the top of this page is pulled from `eew-desc.md` in this folder-->
 
-<big>Together, we can use publicly available data to hold our government and industry accountable.</big>
 
-## Our Goals
+# About EEW
+Environmental Enforcement Watch (EEW) is a collaborative project across Environmental Data & Governance Initiative (EDGI)’s working groups seeking to bring people together from different backgrounds to analyze and engage with the data from the Environmental Protection Agency’s [Enforcement and Compliance History Online](https://echo.epa.gov/) (ECHO) database.
```

Comment:
Instead of adding this (and the next line) here, it should be added in https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/content/about/eew-desc.md | *The contents of `eew-desc.md` are all pulled in here: https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/src/pages/about.js#L16*